.gitlab/ci/global.gitlab-ci.yml

```diff
@@ -29,3 +29,8 @@
   parallel:
     matrix:
       - IMAGE_TAG_SUFFIX: ["", "-python3.7", "-python3.8", "-python3.9"]
+
+.windows:defaults:
```

Comment:
Not removing this only because of https://github.com/meltano/meltano/pull/5967/files